cranelift-codegen/meta/src/isa/x86/recipes.rs

```diff
@@ -367,11 +367,13 @@ pub fn define<'shared>(
     let f_call = formats.by_name("Call");
     let f_call_indirect = formats.by_name("CallIndirect");
     let f_copy_special = formats.by_name("CopySpecial");
+    let f_extract_lane = formats.by_name("ExtractLane"); // TODO this would preferably retrieve a BinaryImm8 format but because formats are compared structurally and ExtractLane has the same structure this is impossible--if we rename ExtractLane, it may even impact parsing
```

Comment:
???

Reply:
That format is useful for more than the ExtractLane instruction; it should probably be named `BinaryImm8` so it can be used for multiple instructions without confusion. But I'm not sure I can change this name without breaking something--what do you think?

Reply:
As far as I know you can pick any name.

Reply:
I was actually just trying to change the name and apparently the name affects all types of files, here's an incomplete list (because I can't get it to build yet):
```
 cranelift-codegen/meta/src/isa/riscv/recipes.rs |  8 ++++----
 cranelift-codegen/meta/src/isa/x86/recipes.rs   | 26 +++++++++++++-------------
 cranelift-codegen/meta/src/shared/formats.rs    |  4 ++--
 cranelift-codegen/src/postopt.rs                |  2 +-
 cranelift-codegen/src/simple_preopt.rs          |  6 +++---
 cranelift-codegen/src/verifier/mod.rs           |  4 ++--
 cranelift-codegen/src/write.rs                  |  4 ++--
 cranelift-reader/src/parser.rs                  | 10 +++++-----
```
It's tied into a bunch of other code so I would prefer leaving this comment in and doing this separately once a few people agree we should change the name.